py3status/modules/google_calendar.py

```diff
@@ -397,7 +406,10 @@ def _build_response(self):
             event_dict["summary"] = event.get("summary")
             event_dict["location"] = event.get("location")
             event_dict["description"] = event.get("description")
-            self.event_urls.append(event["htmlLink"])
+            if self.preferred_event_link == "hangoutLink":
+                self.event_urls.append(event.get("hangoutLink", event.get("htmlLink")))
```

Comment:
Instead of the if / else, couldn't we just try to pick any preferred_event_link and fallback like this?

```suggestion
            self.event_urls.append(event.get(self.preferred_event_link, event.get("htmlLink")))
```

This way we could actually honor any kind of link provided the user knows the key he's interested in (a link to the google api to find out other options in the docstring could be cool)?

Reply:
done, also added additional information about the response status.
Once the PR is merged, I'll offer another change on adding a "?" prefix on upcoming events where you didn't decided if you want to come, or not.